src/main/java/dk/kb/util/InjectionMavenPlugin.java

```diff
@@ -185,7 +106,7 @@ private String createEnumFromYamlValues(YamlResolver yamlInput) {
     private String getSingleValue(YamlResolver yamlInput) {
         try {
             YAML fullYaml = YAML.resolveLayeredConfigs(filePath);
-            return fullYaml.getString(yamlInput.yamlPath);
+            return (String) fullYaml.get(yamlInput.yamlPath);
```

Comment:
Minor thing: There's a dedicated `YAML.getString`-method for retrieving Strings.

Secondarily: If the purpose is to handle all types values (integers, for instance) as Strings, a cast is not the way as it will throw an exception on an `Integer`-object. `toString` works better (and is used by `YAML.getString`).

Reply:
At line 108, ` YAML fullYaml = YAML.resolveLayeredConfigs(filePath);` is called. It will be called for each lookup, which seems a bit excessive. How about creating the `fullYAML` as soon as `setFilePath` is called and then store it for subsequent use?